src/userhistory.py

```diff
@@ -1,33 +1,30 @@
-'''defines user revisions class'''
+'''defines user history class'''
+import datetime
 import requests
-from src.revision import Revision
+from history import History
+from exceptions import BadRequestException
 import mwparserfromhell as mwp
 
 URL = "https://www.wikipedia.org/w/api.php"
 
-class UserRevisions():
-    '''userrevision object parses json user contributions '''
+class UserHistory():
```

Comment:
derive this class from the base class by putting the base class name in the parens